apps/authentications/src/app/dtos/dto.ts

```diff
@@ -29,6 +29,12 @@ export class GuestLoginDto {
   @IsNotEmpty()
   @ApiProperty()
   deviceUUID: string;
+
+  @IsNotEmpty()
+  @ApiProperty()
+  clientInfo: {
+    ipAddress: string;
+  };
```

Comment:
The `clientInfo` can be string, number, or any type. So, we should create a new class for validating the `clientInfo` property.

```ts
class ClientInfoDto {
    @IsString()
    ipAddress: string;
};
```


```suggestion
  @ApiProperty()
  @ValidateNested()
  @Type(() => ClientInfoDto)
  clientInfo: ClientInfoDto;
```